curriculum/structure/blocks/lab-player-interface.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "Build a Player Interface",
+  "isUpcomingChange": true,
+  "dashedName": "lab-player-interface",
+  "helpCategory": "Python",
+  "blockLayout": "link",
+  "challengeOrder": [
+    {
+      "id": "68e2c945cc1d8e778152be31",
+      "title": "Build a Player Interface"
+    }
+  ],
+  "blockType": "lab",
```

```suggestion
  "blockLabel": "lab",
```

important to do by next time you merge main / before merging the PR